kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/job/PipelineJobCenter.java

```diff
@@ -59,14 +63,20 @@ public static boolean isJobExisting(final String jobId) {
      * Stop job.
      *
      * @param jobId job id
+     * @param isAsync is async to stop
      */
-    public static void stop(final String jobId) {
+    public static void stop(final String jobId, final boolean isAsync) {
```

Comment:
It's better not add new parameter, since:
1, Most of cases use `true` for `isAsync` param;
2, We could do `persistEphemeralChildrenNode` just on where it call this `stop` and use `false` for `isAysnc`;